configure.ac

```diff
@@ -68,6 +68,7 @@ AC_PATH_TOOL(GCOV, gcov)
 AC_PATH_PROG(LCOV, lcov)
 AC_PATH_PROG(JAVA, java)
 AC_PATH_PROGS([PYTHON], [python3 python2.7 python2 python])
+AC_PATH_PROGS([PYTHON2], [python2.7 python2 python])
```

Comment:
Yes. Totally agree, ... I'm testing now the solution from @theuni (https://github.com/bitcoin/bitcoin/issues/8120#issuecomment-223691744).